samples/sample_common/include/sample_utils.h

```diff
@@ -299,6 +299,32 @@ struct IsPairedMfxExtBuffer : std::false_type {};
 template <>
 struct IsPairedMfxExtBuffer<mfxExtAVCRoundingOffset> : std::true_type {};
 
+template <typename R>
+struct ExtParamAccessor
```

Comment:
I don't get the idea. Why do we need it? Ok, it works for every class that has those two fields... But already can access them directly, without any additional layer. could you point to place where we need such kind of accessor?

Reply:
See mfxFrameSurface1 case